hyp3_autorift/vend/testautoRIFT.py

```diff
@@ -996,8 +996,8 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
                         'processing_date_img2': slave_split[4][0:8],
                         'row_img1': np.float64(master_split[2][3:6]),
                         'row_img2': np.float64(slave_split[2][3:6]),
-                        'satellite_img1': np.float64(master_split[0][2:4]),
-                        'satellite_img2': np.float64(slave_split[0][2:4]),
+                        'satellite_img1': master_split[0][2:4].lstrip('0'),
```

Comment:
I'm assuming we're fine with this because this pain-point throughout auto rift, but if the name schema ever changes, this will break. Just noting for posterity.

Reply:
Fortunately, the Landsat naming convention has been static since Landsat-1, so I think we're okay here.